shared/fs/top-bar/index.js

```diff
@@ -38,7 +39,7 @@ const TopBar = (props: Props) =>
       {Types.getPathLevel(props.path) === 3 && <SyncToggle tlfPath={props.path} />}
       <Kb.Box style={styles.flex} />
       <Loading path={props.path} />
-      <Sort path={props.path} />
+      {props.mode !== 'offline' && <Sort path={props.path} />}
```

Comment:
It'd be nice to avoid this sort of plumbing. Perhaps [this](https://github.com/keybase/client/pull/16848/files#diff-088e08f0ca0e38cd16c0b120a3ebc514R25) can be moved into `Constants`, and we can use the same logic in `sort-container.js` where it also deals with `isEmpty`?

Reply:
Good point; changed.